mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -977,6 +990,9 @@ func flattenUpdatePolicy(updatePolicy *computeBeta.InstanceGroupManagerUpdatePol
 		}
 		up["min_ready_sec"] = updatePolicy.MinReadySec
 		up["minimal_action"] = updatePolicy.MinimalAction
+		<% if version == "beta" -%>
+		up["most_disruptive_allowed_action"] = updatePolicy.MostDisruptiveAllowedAction
```

Comment:
Does `most_disruptive_allowed_action` come back from the API? I have seen similar fields not be returned as they only apply to the update itself

Reply:
Is a default value set by the API for this is the user doesn't set it? We will need to test not setting it in a config so that we can see the API's behavior here and make sure we handle it

Reply:
> Does `most_disruptive_allowed_action` come back from the API? I have seen similar fields not be returned as they only apply to the update itself

Yes, This values is stored in IGM.updatePolicy and will be returned in IGM.get response.



> Is a default value set by the API for this is the user doesn't set it? We will need to test not setting it in a config so that we can see the API's behavior here and make sure we handle it

 No, if user doesn't specify there are no default value. IGM.get will return UpdatePolicy without explicit mostDisruptiveAllowedAction (effectively it will behave as REPLACE).